multi-agent/adapters/outbound/channels/local/__init__.py

```
@@ -1,4 +1,10 @@
 from .channel import LocalSessionChannel, LocalSessionChannelReader
+from .input_channel import LocalInputCapableChannel
 from .factory import LocalChannelFactory
 
-__all__ = ["LocalSessionChannel", "LocalSessionChannelReader", "LocalChannelFactory"]
+__all__ = [
+    "LocalSessionChannel",
+    "LocalSessionChannelReader",
+    "LocalInputCapableChannel",
+    "LocalChannelFactory",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)